ocs_ci/framework/tests/test_config.py

```diff
@@ -45,6 +45,26 @@ def test_layered_conf(self):
         assert framework.config.RUN["client_version"] == "2"
         assert framework.config.DEPLOYMENT["installer_version"] == "1"
 
+    def test_custom_conf_multicluster(self):
```

Comment:
+1 for unit testing this functionality. Can we add others? I can see tests being created for resetting, updating, resetting context, etc. The more we cover the more confident we can be that we aren't breaking this when changes are introduced.